### PR TITLE
[MIG] account_chatter: Migration to 16.0 T#66175

### PR DESCRIPTION
**Pull request by VEscaname**
Adding groups to inherited views is no longer allowed. Because of this, the views in the module were refactored in order for them to continue functioning properly.

### CODE REVIEW

**Comment by VEscaname** [comment]
@luisg123v could you review this please?

File: account_chatter/__manifest__.py [hunk 1]
```diff
@@ -1,9 +1,6 @@
-# Copyright 2020 Vauxoo
-# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
```
Comment (luisg123v) [inline account_chatter/__manifest__.py]: Apply this change in v15.

BTW, I noticed they were not removed from file `models/__init__.py`, along with all non-migration changes.

File: account_chatter/models/account_account.py [hunk 2]
```diff
@@ -1,11 +1,7 @@
-# Copyright 2020 Vauxoo
-# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
-
 from odoo import fields, models
 
 
 class AccountAccount(models.Model):
-
```
Comment (luisg123v) [inline account_chatter/models/account_account.py]: Apply change in previous version

File: account_chatter/views/account_account_views.xml [hunk 3]
```diff
@@ -1,29 +1,24 @@
-<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" encoding="UTF-8" ?>
 <odoo>
 
-    <record id="view_account_form_show_chatter" model="ir.ui.view">
+     <record id="view_account_form_show_chatter" model="ir.ui.view">
```
Comment (luisg123v) [inline account_chatter/views/account_account_views.xml]: Wrong indentation

File: account_chatter/views/account_move_views.xml [hunk 4]
```diff
@@ -1,30 +1,23 @@
-<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" encoding="UTF-8" ?>
 <odoo>
 
-    <record id="view_move_form_show_chatter" model="ir.ui.view">
+    <record id="view_move_form" model="ir.ui.view">
         <field name="name">account.move.form.inherit.account.chatter.show</field>
         <field name="model">account.move</field>
         <field name="inherit_id" ref="account.view_move_form" />
-        <field name="groups_id" eval="[(4, ref('account_chatter.group_show_account_chatter_notifications'))]" />
```
Comment (luisg123v) [inline account_chatter/views/account_move_views.xml]: Are you sure this doesn't work anymore?

Is ee the field [still exists in v16](https://github.com/odoo/odoo/blob/79dc6384329b1e8bd2338160d62028ac0c1cdce4/odoo/addons/base/models/ir_ui_view.py#L263), so it should be possible

File: account_chatter/views/account_account_views.xml [hunk 5]
```diff
@@ -1,29 +1,24 @@
...
+            <xpath expr="//div[hasclass('oe_chatter')]" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
             <xpath expr="//sheet" position="after">
-                <div class="oe_chatter">
-                    <field name="message_follower_ids" />
-                    <field name="activity_ids" />
-                    <field name="message_ids" />
-                </div>
+                <t groups="account_chatter.group_show_account_chatter_notifications">
```
Comment (luisg123v) [inline account_chatter/views/account_account_views.xml]: Does this work?

Take into account t directives only work in QWeb templates, they dont in standard views like form or tree view.

How is this element rendered? You may check by accessing Developer tools -> Fields view get, which shows you the rendered XML.

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by VEscaname** [comment]
@luisg123v could you review the changes again please?

**Comment by luisg123v** [comment]
@VEscaname missing new commits from 15.0

File: account_chatter/views/account_account_views.xml [hunk 6]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <record id="view_account_form_show_chatter" model="ir.ui.view">
```
Comment (luisg123v) [inline account_chatter/views/account_account_views.xml]: Since this is now a standard inheritance (without groups), ID should match inherited view

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by VEscaname** [comment]
@luisg123v could you review once again?

**Review by luisg123v — APPROVED** [review]
LGTM :+1: 

I'll merge after CI is initialized

CC @desdelinux

**Pull request merged**